diff --git a/Ortho4XP_v130.py b/Ortho4XP_v130.py
index 58b10c9..7c5c34e 100755
--- a/Ortho4XP_v130.py
+++ b/Ortho4XP_v130.py
@@ -11,6 +11,7 @@ import O4_Vector_Map as VMAP
 import O4_Mesh_Utils as MESH
 import O4_Mask_Utils as MASK
 import O4_Tile_Utils as TILE
+import O4_Overlay_Utils as OVL
 import O4_GUI_Utils as GUI
 import O4_Config_Utils as CFG  # CFG imported last because it can modify other modules variables
 
@@ -68,6 +69,7 @@ if __name__ == '__main__':
             MESH.build_mesh(tile)
             MASK.build_masks(tile)
             TILE.build_tile(tile)
+            OVL.build_overlay(lat,lon)
             print("Bon vol!")
         except:
             print("Crash!")
